staging/src/k8s.io/apimachinery/pkg/api/operation/operation.go

```diff
@@ -41,12 +39,22 @@ type Operation struct {
 	// resource first began using the feature.
 	//
 	// Unset options are disabled/false.
-	Options sets.Set[string]
+	Options []string
```

Comment:
In this cycle I saw some places that passed non-bool options.  E.g. NetworkPolicyValidationOptions has a list of CIDRs to allow even if invalid.  This is part of ratcheting, but I wonder if options also needs to handle that?  I guess we can get there when we get there.  FYI until then.